src/frontend/utils/dato/dato.ts

```diff
@@ -105,3 +105,6 @@ export const parseFraOgMedDato = (fom: IsoDatoString | undefined) =>
         isoString: fom,
         fallbackDate: tidenesMorgen,
     });
+
+export const erSammeFom = (dato1?: IsoDatoString, dato2?: IsoDatoString): boolean =>
+    isSameDay(parseFraOgMedDato(dato1), parseFraOgMedDato(dato2));
```

Comment:
Samme gjelder denne, hvis den kun brukes i den ene filen tror jeg at jeg foretrekker å ha denne funksjonen i samme fil hvor den blir brukt 🤔

Reply:
Flytter denne, men beholder de andre siden de nå brukes to steder cdff737